### PR TITLE
Add SIMD acronym explanation and bibliography entry

### PR DESCRIPTION
**Pull request by mhoemmen**
- [x] Add Note explaining the SIMD acronym
- [x] Add bibliography entry to Flynn's original paper explaining the SISD, SIMD, MISD, MIMD taxonomy
- [x] Cite that paper in the Note

### CODE REVIEW

File: source/numerics.tex [hunk 1]
```diff
@@ -16083,7 +16083,12 @@
 \rSec2[simd.general]{General}
 
 \pnum
-Subclause \ref{simd} defines data-parallel types and operations on these types.
+Subclause \ref{simd}
+\begin{note}
```
Comment (jensmaurer) [inline source/numerics.tex:16086]: Since notes end up as separate paragraphs, having a note in the middle of a sentence feels very much sub-optimal. Since this is contextual information (wider context), a footnote feels appropriate for that.

**Review by jensmaurer — CHANGES_REQUESTED** [review]
Please rephrase the commit descriptions: https://github.com/cplusplus/draft/wiki/Commit-message-format

File: source/numerics.tex (hunk 1 above)
Comment (mhoemmen) [reply source/numerics.tex:16086]: Thanks Jens!  I'll combine the two Notes into one.

Comment (mhoemmen) [reply source/numerics.tex:16087]: @jensmaurer @tkoeppe @jwakely I've consolidated the two Notes into one as requested.  Please let me know if you would like any additional changes.  Thanks!

File: source/back.tex [hunk 2]
```diff
@@ -77,15 +77,19 @@
...
+  J.\ Du Croz, A.\ Greenbaum, S.\ Hammarling, A.\ McKenney, and D.\ Sorensen,
   \doccite{LAPACK Users' Guide, Third Edition}.
   SIAM, Philadelphia, PA, USA, 1999.
 \bibitem{blas-std}
-  L. Susan Blackford, Ames Demmel, Jack Dongarra, Iain Duff, Sven Hammarling,
+  L.\ Susan Blackford, James Demmel, Jack Dongarra, Iain Duff, Sven Hammarling,
   Greg Henry, Michael Heroux, Linda Kaufman, Andrew Lumbsdaine, Antoine Petitet,
-  Roldan Pozo, Karin Remington, R. Client Whaley
+  Roldan Pozo, Karin Remington, and R.\ Client Whaley,
   \doccite{An Updated Set of Basic Linear Algebra Subprograms (BLAS)}.
```
Comment (tkoeppe) [inline source/back.tex:87]: Oh god, could you send those existing fixes as a separate PR please? That's horrendous, well spotted!

File: source/numerics.tex [hunk 3]
```diff
@@ -16085,7 +16085,9 @@
 \pnum
 Subclause \ref{simd} defines data-parallel types and operations on these types.
 \begin{note}
-The intent is to support acceleration through data-parallel execution resources
+SIMD stands for ``Single Instruction Stream -- Multiple Data Stream'';
```
Comment (tkoeppe) [inline source/numerics.tex:16088]: Maybe make this sentence the second one?

File: source/back.tex (hunk 2 above)
Comment (mhoemmen) [reply source/back.tex:87]: Thanks! : - )  I'll move those fixes into a separate PR.

Comment (mhoemmen) [reply source/back.tex:87]: I've removed these fixes from this PR, and moved them to PR https://github.com/cplusplus/draft/pull/7507 .  Thanks! : - )

File: source/numerics.tex (hunk 3 above)
Comment (mhoemmen) [reply source/numerics.tex]: Done; thanks! : - )

**Comment by mhoemmen** [comment]
@jensmaurer wrote:
> Please rephrase the commit descriptions: https://github.com/cplusplus/draft/wiki/Commit-message-format

Thanks Jens!  I'll edit the commit messages to correspond to the "Editorial commits" format.

**Review by jensmaurer — CHANGES_REQUESTED** [review]
Squash all commits, fix the commit description to read something like

[simd.general] Explain SIMD acronym and add bibliography entry

and force-push.

**Comment by mhoemmen** [comment]
@jensmaurer wrote:
> Squash all commits, fix the commit description to read something like [simd.general] Explain SIMD acronym and add bibliography entry and force-push.

Done!  There are two commits now, one for the [bibliography] change and one for the [simd.general] change.  Please let me know if you would like those two changes in one commit.  Thanks!

**Review by jensmaurer — APPROVED** [review]

**Comment by tkoeppe** [comment]
Let me squash them, so that the reference is never "unused. Thank you!
